.github/workflows/continuous-integration.yml

```diff
@@ -639,6 +639,14 @@ jobs:
         with:
           platforms: ${{ needs.buildx.outputs.platforms }}
 
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          # Disable setup-go caching. Cache is better handled by the golangci-lint action
+          cache: false
```

Comment:
Shouldn't we drop these two lines? They are meant only for the linter step. The olm builder would benefit from the cache.